test/globalize/attributes_test.rb

```diff
@@ -5,11 +5,11 @@ class AttributesTest < MiniTest::Spec
 
   describe 'translated attribute reader' do
     it 'is defined for translated attributes' do
-      if ::ActiveRecord::VERSION::STRING < "5.0"
-        assert_equal Post.new.respond_to?(:title), false
-      else
-        assert_equal Post.new.respond_to?(:title), true
-      end
+      assert Post.new.respond_to?(:title)
+    end
+
+    it 'Post#columns does not include translated attributes' do
```

Comment:
@akfernun we don't need a version check?

Reply:
@parndt, I don't think so. In Rails 4.2, calling the `attribute` method results in the column being added to the class's `.columns` hash even though it's not an actual column in the database. In Rails 5.2, calling the `attribute` method **does not** result that column being added to the `.columns` hash. Since we added the version check in the code, we can be confident that no matter what version of Rails we're using, calling `.columns` will **never** result in a translated attribute ending up in the `.columns` hash.